dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -95,11 +166,7 @@ window.Dashboard = {
 	__handleAppEvent: function (event) {
 		switch (event.name) {
 			case "CONFIG_READY":
-				if (window.location.protocol === "http:" && this.config.INSTALL_CERT) {
-					window.location.href = window.location.href.replace("http", "https");
```

Comment:
I'm not sure this should be removed @jvatic . When requesting `http://dashboard.FLYNN_CLUSTER` I always get redirected to `/installcert` although the certificate is installed correctly. When requesting `https://dashboard.FLYNN_CLUSTER` everything seems to be working fine.